tests/test_mkieee.F90

```diff
@@ -9,7 +9,7 @@ program test_mkieee
 #ifdef KIND_4
   integer, parameter :: num = 7
   real :: rieee(num)
-  real :: a(num) = (/ 4.3000000, 5.6000000, 1.6700000, 2.3300000, -4.3000000, 0.3300000, -400.0000000 /)
```

Comment:
Instead of taking out the test for -400.00000 and adding one for 0, just add one for 0.

In other words, leave the existing test value and add a new value to the test array.

Reply:
Can you make this change and then we can merge this?